[NistApiMirrorTask] An unexpected error occurred while mirroring the contents of the National Vulnerability Database

### Current Behavior

Our instance of dependency track NVD has not been updated since the 15th Feb 2025

When looking at the api container logs we have these errors:

`
2025-02-17 05:51:22,188 ERROR [NistApiMirrorTask] An unexpected error occurred while mirroring the contents of the National Vulnerability Database 
io.github.jeremylong.openvulnerability.client.nvd.NvdApiException: Failed to parse NVD data
at io.github.jeremylong.openvulnerability.client.nvd.NvdCveClient._next(NvdCveClient.java:363)
at io.github.jeremylong.openvulnerability.client.nvd.NvdCveClient.next(NvdCveClient.java:331)
at org.dependencytrack.tasks.NistApiMirrorTask.inform(NistApiMirrorTask.java:156)
at alpine.event.framework.BaseEventService.lambda$publish$0(BaseEventService.java:110)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base/java.lang.Thread.run(Unknown Source)
Caused by: com.fasterxml.jackson.databind.exc.ValueInstantiationException: Cannot construct instance of io.github.jeremylong.openvulnerability.client.nvd.CvssV4Data$ModifiedCiaType, problem: SAFETY
at [Source: REDACTED (StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION disabled); line: 1, column: 53619] (through reference chain: io.github.jeremylong.openvulnerability.client.nvd.CveApiJson20["vulnerabilities"]->java.util.ArrayList[24]-io.github.jeremylong.openvulnerability.client.nvd.DefCveItem["cve"]-io.github.jeremylong.openvulnerability.client.nvd.CveItem["metrics"]-io.github.jeremylong.openvulnerability.client.nvd.Metrics["cvssMetricV40"]->java.util.ArrayList[0]-io.github.jeremylong.openvulnerability.client.nvd.CvssV4["cvssData"]-io.github.jeremylong.openvulnerability.client.nvd.CvssV4Data["modifiedSubsequentSystemIntegrity"])
`

We enabled the NVD mirroring via API using the default endpoint: https://services.nvd.nist.gov/rest/json/cves/2.0
When manually hitting this endpoint either via postman or curl, the response does not have cvssMetricV40 element

### Steps to Reproduce

1. Enable National Vulnerability Database mirroring
2. Enable mirroring via API with the default endpoint: https://services.nvd.nist.gov/rest/json/cves/2.0
3. Wait for mirroring to occur (once per day) or force it by restarting the api container
4. Error occur when the api container runs the NistApiMirrorTask

### Expected Behavior

The NistApiMirrorTask should not fail to parse NVD data

### Dependency-Track Version

4.12.4

### Dependency-Track Distribution

Container Image

### Database Server

PostgreSQL

### Database Server Version

16.2

### Browser

Google Chrome

### Checklist

- [x] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#filing-issues)
- [x] I have checked the [existing issues](https://github.com/DependencyTrack/dependency-track/issues) for whether this defect was already reported

--- a/pom.xml
+++ b/pom.xml
@@ -112,7 +112,7 @@
         <lib.lucene.version>8.11.4</lib.lucene.version>
         <lib.maven-artifact.version>3.9.9</lib.maven-artifact.version>
         <lib.mockserver-netty.version>5.15.0</lib.mockserver-netty.version>
-        <lib.open-vulnerability-clients.version>7.2.1</lib.open-vulnerability-clients.version>
+        <lib.open-vulnerability-clients.version>7.2.2</lib.open-vulnerability-clients.version>
         <lib.packageurl.version>1.5.0</lib.packageurl.version>
         <lib.pebble.version>3.2.2</lib.pebble.version>
         <lib.protobuf-java.version>4.28.2</lib.protobuf-java.version>

